Update/kage_010.txt

```diff
@@ -2452,7 +2452,7 @@ void main()
 //rそれとも、小説で悲惨な末路が待っている時に、途中で読むのを止めるのと同じように、全て、『なかった』ことにしてしまうってのは？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "それとも、小説で悲惨な末路が待っている時に、途中で読むのを止めるのと同じように、全て、『なかった』ことにしてしまうってのは？",
-			NULL, "When you know a novel is leading to a miserable ending, can you just stop reading part way through and declare that the bad ending <i>doesn't exist?</i>", Line_Normal);
+			NULL, "Or make it so that none of it ever <i>happened</i>, like a novel with a tragic ending you stop reading part way through?", Line_Normal);
```

Comment:
```suggestion
			NULL, "Or make it so that none of it ever <i>happened</i>, like stopping a novel part way through when you know it has a tragic ending waiting for you?", Line_Normal);
```